debian/control

```diff
@@ -67,7 +67,7 @@ Build-Depends: debhelper-compat (= 13),
                python3-pyelftools [amd64 i386 arm64 armhf riscv64] <!stage1>,
                python3-pyparsing <!nocheck>,
                python3-evdev:native <!nocheck>,
-               tzdata-legacy <!nocheck> | tzdata (<< 2025a-2) <!nocheck>,
+               tzdata-legacy <!nocheck> | tzdata <!nocheck>,
```

Comment:
I don't remember all the details, but there's definitely a point to the constraint. This is something that would need review if the base version is ever upgraded from bookworm and `tzdata-legacy` is available.